src/utils.py

```diff
@@ -49,3 +49,21 @@ def storeImage(im, name):
     # convert and store
     im_c = im_c.astype(np.uint8)
     IMAGES_STACK[name] = im_c
+
+
+def dictFromList(liste):
```

Comment:
ok right, do you know if it is possible to add this checkup to pre-commit?

Reply:
Method names in PyQt are written in camel case, ex:
`def mouseReleaseEvent(self, event)`
So which case should I use for methods in class?

Reply:
It is known that PyQt does not follow PEP for its function name, also because Qt is written in C++, then I think they kept names from Qt for PyQt.
In our case, we must just choose if we want to follow this PEP8 recommendation or not. I will see if it is possible to add this check in pre-commit!

Reply:
Thx! I think we should keep camel case for pyqt inherited classes and snake case for any other functions/variables/classes

Reply:
Ok, I agree with that :+1: